library-share-via-example/capsule.bxb

```diff
@@ -1,8 +1,9 @@
 capsule {
   id (example.shareViaExample)
-  version (1.0.0)
+  version (1.1.0)   // JSRS migration VCAP-24612
   format (3)
   runtime-version (7) {
```

Comment:
while we wait on the library deprecation decision, let's update this to runtime 9